ports/esp32/machine_pwm.c

```diff
@@ -205,18 +219,15 @@ STATIC void configure_channel(machine_pwm_obj_t *self) {
 }
 
 STATIC void set_freq(machine_pwm_obj_t *self, unsigned int freq, ledc_timer_config_t *timer) {
-    // Even if the timer frequency is already set,
-    // the set_duty_x() is required to reconfigure the channel duty anyway
     if (freq != timer->freq_hz) {
-        PWM_DBG("set_freq(%d)", freq)
 
         // Find the highest bit resolution for the requested frequency
         unsigned int i = LEDC_APB_CLK_HZ; // 80 MHz
         if (freq < EMPIRIC_FREQ) {
             i = LEDC_REF_CLK_HZ; // 1 MHz
         }
 
-        #if 1
+        #if 0
```

Comment:
Can this small block of code be removed now that it's unused (the 2 lines below here)?

Reply:
Will remove after using ESP-IDF5.